{"identifier":{"url":"doc:\/\/Discord\/documentation\/Discord\/DiscordGatewayCloseReason\/authenticationFailed","interfaceLanguage":"swift"},"hierarchy":{"paths":[["doc:\/\/Discord\/documentation\/Discord","doc:\/\/Discord\/documentation\/Discord\/DiscordGatewayCloseReason"]]},"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"text":"static","kind":"keyword"},{"text":" ","kind":"text"},{"text":"let","kind":"keyword"},{"text":" ","kind":"text"},{"text":"authenticationFailed","kind":"identifier"},{"text":": ","kind":"text"},{"text":"DiscordGatewayCloseReason","preciseIdentifier":"s:7Discord0A18GatewayCloseReasonV","kind":"typeIdentifier","identifier":"doc:\/\/Discord\/documentation\/Discord\/DiscordGatewayCloseReason"}],"platforms":["Linux"],"languages":["swift"]}]}],"kind":"symbol","schemaVersion":{"patch":0,"major":0,"minor":2},"sections":[],"abstract":[{"text":"We failed to authenticate with Discord.","type":"text"}],"variants":[{"paths":["\/documentation\/discord\/discordgatewayclosereason\/authenticationfailed"],"traits":[{"interfaceLanguage":"swift"}]}],"metadata":{"title":"authenticationFailed","modules":[{"name":"Discord"}],"fragments":[{"kind":"keyword","text":"static"},{"kind":"text","text":" "},{"kind":"keyword","text":"let"},{"kind":"text","text":" "},{"kind":"identifier","text":"authenticationFailed"},{"kind":"text","text":": "},{"kind":"typeIdentifier","preciseIdentifier":"s:7Discord0A18GatewayCloseReasonV","text":"DiscordGatewayCloseReason"}],"symbolKind":"property","role":"symbol","roleHeading":"Type Property","externalID":"s:7Discord0A18GatewayCloseReasonV20authenticationFailedACvpZ"},"references":{"doc://Discord/documentation/Discord":{"role":"collection","kind":"symbol","type":"topic","identifier":"doc:\/\/Discord\/documentation\/Discord","abstract":[],"title":"Discord","url":"\/documentation\/discord"},"doc://Discord/documentation/Discord/DiscordGatewayCloseReason/authenticationFailed":{"identifier":"doc:\/\/Discord\/documentation\/Discord\/DiscordGatewayCloseReason\/authenticationFailed","kind":"symbol","url":"\/documentation\/discord\/discordgatewayclosereason\/authenticationfailed","fragments":[{"text":"static","kind":"keyword"},{"text":" ","kind":"text"},{"text":"let","kind":"keyword"},{"text":" ","kind":"text"},{"text":"authenticationFailed","kind":"identifier"},{"text":": ","kind":"text"},{"text":"DiscordGatewayCloseReason","preciseIdentifier":"s:7Discord0A18GatewayCloseReasonV","kind":"typeIdentifier"}],"type":"topic","abstract":[{"text":"We failed to authenticate with Discord.","type":"text"}],"role":"symbol","title":"authenticationFailed"},"doc://Discord/documentation/Discord/DiscordGatewayCloseReason":{"role":"symbol","type":"topic","fragments":[{"kind":"keyword","text":"struct"},{"kind":"text","text":" "},{"kind":"identifier","text":"DiscordGatewayCloseReason"}],"identifier":"doc:\/\/Discord\/documentation\/Discord\/DiscordGatewayCloseReason","url":"\/documentation\/discord\/discordgatewayclosereason","navigatorTitle":[{"kind":"identifier","text":"DiscordGatewayCloseReason"}],"kind":"symbol","title":"DiscordGatewayCloseReason","abstract":[{"text":"Represents the reason a gateway was closed.","type":"text"}]}}}